### PR TITLE
COMMERCE-6474 Batch delete operation fails when deleting Products

### PR DESCRIPTION
**Pull request by matijapetanjek**
Hi @andrea-ale-sbarra, can you please check if this is the valid way to fix batch delete operation for products (meaning if it's OK to change the code directly or is it possible to do it via restBuilder and specification). Thank you! /cc @igorbeslic

### CODE REVIEW

**Comment by matijapetanjek** [comment]
ci:test:sf

**Comment by matijapetanjek** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/BaseProductResourceImpl.java [hunk 1]
```diff
@@ -411,7 +411,7 @@ public void delete(
 		throws Exception {
 
 		for (Product product : products) {
-			deleteProduct(product.getId());
+			deleteProduct(product.getProductId());
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/BaseProductResourceImpl.java:414]: Hi @matijapetanjek that is not allowed, you should never change base classes, I think you can override that method or you can ask for support on t-headless channel on Slack

**Comment by matijapetanjek** [comment]
ci:test:sf

**Comment by matijapetanjek** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:forward

**Pull request closed without merging**